### PR TITLE
LPD-40551 Adapt Playwright tests

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:page-management-playwright

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ShortcutManager.js [hunk 1]
```diff
@@ -251,6 +251,8 @@ export default function ShortcutManager() {
 			copy: {
 				action: copy,
 				canBeExecuted: () =>
+					!isEditingEditableField() &&
+					window.getSelection().type !== 'Range' &&
```
Comment (georgel-pop-lr) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ShortcutManager.js:255]: I think we could get this to a method like isEditingEditableField

**Comment by georgel-pop-lr** [comment]
Good job @sandrodw3 I added only one comment that I think we can improve. 

Regards.

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ShortcutManager.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ShortcutManager.js]: Done!

**Comment by sandrodw3** [comment]
ci:test:page-management-playwright

**Comment by sandrodw3** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**